fabric_rti_mcp/services/kusto/kusto_service.py

```diff
@@ -554,3 +554,55 @@ def kusto_get_shots(
     """
 
     return _execute(kql_query, cluster_uri, database=database, client_request_properties=client_request_properties)
+
+def rootcause_analysis_query(
```

Comment:
I don't see why this is called root cause analysis, the query is not specific.
Also, I wouldn't hard-code queries here.